src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -158,9 +159,16 @@
                           (= status-code 502) (reset! maintenance-mode? true)
                           (contains? (set (range 500 600)) status-code) (reset! server-down? true)))
                       ;; Handle auth token expiration
-                      (when (and (= status-code 401) (not ignore-auth-expiration?))
-                        (auth-expiration-handler))
-                      (on-done m))))))
+                      (if (and (= status-code 401) (not ignore-auth-expiration?))
+                        (do
+                          (swap! pending-calls conj [path arg-map
+                                                     :service-prefix service-prefix
+                                                     :ignore-auth-expiration true])
```

Comment:
I don't believe you :)

In fact, both don't work. The only one that works is no brackets and spelling `:ignore-auth-expiration?` (with the question mark) correctly.